txpool/txpool.go

```diff
@@ -467,6 +468,11 @@ func (p *TxPool) processEvent(event *blockchain.Event) {
 // validateTx ensures the transaction conforms to specific
 // constraints before entering the pool.
 func (p *TxPool) validateTx(tx *types.Transaction) error {
+	// FIXME: Accepts only legacy transaction only for now
+	if tx.Type != types.TxTypeLegacy {
+		return ErrInvalidTxType
+	}
```

Comment:
I have no idea, leftover?

Reply:
No, we need to fix when we add new type transaction besides StateTransaction